matter/efr32/efr32mg24/BRD4187C/autogen/sl_event_handler.h

```diff
@@ -7,5 +7,8 @@ void sl_driver_init(void);
 void sl_service_init(void);
 void sl_stack_init(void);
 void sl_internal_app_init(void);
+#ifdef CHIP_917
+void sl_iostream_init_instances(void);
```

Comment:
BTW I took it  out from here because the prototype is also defined in `autogen/sl_iostream_init_instances.h. ` Thats a SLC issue you can remove this one

Reply:
removed